### DISCUSSION TITLE
Feature suggestion: Changed Palette theme could affect blog view?

### DISCUSSION THREAD

**Discussion by bitec0de** [AprilSylph/XKit-Rewritten#646] [Suggestion Box]
Viewing a blog from the dashboard using a URL like `www.tumblr.com/blog/view/< account-name-here >`, the styling was always black text, white background, ignoring my dark theme settings (chosen from the main Tumblr > Account menu).

The new Themed Posts setting "Enable theming on the blog view" helps, but many blogs are still illegibly bright due to their owners' preferences.

Due to visual impairment, i'd really like it if i never had to read text against a white background. Adding a toggle checkbox to Themed Posts to use whichever of a blog's colors is darker as the background color would work, or using the user's selected Palette if there's a way to detect that.

**Comment by bitec0de** [comment]
I made this simple change to a fork https://github.com/bitec0de/XKit-Rewritten and it works much better for me!

Not sure how pull requests work or what the next step is.

**Comment by AprilSylph** [comment]
This is already a Tumblr feature that is rolling out in dashboard settings, currently out to 70% of users.

![image](https://user-images.githubusercontent.com/28949509/176669530-51ece0d4-c831-4e46-917d-b54330028e1a.png)

**Reply by bitec0de** [reply]
Oh, great, i was puzzled this wasn't the behaviour already, sorry for bothering you!

**Reply by marcustyphoon** [reply]
This feature was just announced as released on the changes blog, and I have the toggle, but it doesn't seem to me like it addresses the user's feature request?

**Reply by bitec0de** [reply]
The new tumblr toggle works exactly how i'd wanted this suggested XKit feature to; my idea about selecting darker colours from a blog's style to be the background just seemed a simple way to approach the same result by modifying an already existing feature.

**Comment by arcanetrivia** [comment]
With the toggle off, it uses my current selected palette when I look at a post in blog view, whether that be dark theme or whatever, just as described. Is it not working that way for you, marcustyphoon? Seems to me like it fits the bill of OP's request for "using the user's selected Palette if there's a way to detect that." (although maybe Themed Posts is conflicting with that setting somehow? I'm not using Themed Posts)

**Comment by marcustyphoon** [comment]
Oh, awesome, never mind then 👍
